templates/CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.tpl

```diff
@@ -38,7 +38,9 @@
 CRM.$(function($) {
 
   function checkPeriod() {
-    var speriod = $('#prior_financial_period').val();
+    var m = $('#prior_financial_period_M').val();
+    var y = $('#prior_financial_period_Y').val();
+    var speriod = m + '-' + y;
```

Comment:
Again, this order should not be hardcoded as it needs to be localized via the Month and Year setting. Could you update to use it, and review for any other hardcoding? Thx.